mm2src/coins/utxo/utxo_withdraw.rs

```diff
@@ -26,6 +26,18 @@ use utxo_signer::sign_params::{OutputDestination, SendingOutputInfo, SpendingInp
 use utxo_signer::{with_key_pair, UtxoSignTxError};
 use utxo_signer::{SignPolicy, UtxoSignerOps};
 
+/// Fee (in satoshis) applied to withdrawal transactions
+/// that have a high priority, indicating a need for faster confirmation.
+pub const HIGH_TX_FEE: u16 = 4500;
```

Comment:
Wondering, how were these consts obtained?
I believe this is 4500 per vKByte, that is 4,4 sat/vB. Isn't it too low?
I can see, mempool.space shows at the moment high priority fee as 49...74 sat/vB (that would cost 4,2..6,2USD for a common segwit tx of 140vB).

Also, do we consider using a provider for obtaining tx fee or calculate it based on history?

Reply:
I could have used `estimate_fee_sat` with specified `n_block` to estimating the fee but I ended up using hardcoded values because the rpc never actually works for almost all utxo coins 

https://github.com/KomodoPlatform/komodo-defi-framework/blob/b0fd99e8406e67ea06435dd028991caa5f522b5c/mm2src/coins/utxo/rpc_clients.rs#L368-L374

Reply:
do you have some ideas for this @dimxy